server/Server/PageCreator.ts

```diff
@@ -6,6 +6,11 @@ import { Request, Response } from "express"
 import * as CONTRACT from "../public/contract/contract.json";
 
 class PageCreator {
+
+    private static formatPrice(price: Number) {
+        return price.toFixed(2)
```

Comment:
Mettici anche la divisione per 100 no?

Poi `statusConverter` e `timeConverter`, non sono statici per coerenza falli tutti uguali (in un modo o nell'altro)